src/Disks/S3/registerDiskS3.cpp

```diff
@@ -161,7 +161,7 @@ std::unique_ptr<DiskS3Settings> getSettings(const Poco::Util::AbstractConfigurat
 
     return std::make_unique<DiskS3Settings>(
         getClient(config, config_prefix, context),
-        std::move(rw_settings),
+        rw_settings,
```

Comment:
What's wrong here?

Reply:
> What's wrong here?
Since  'rw_settings' is  trivially-copyable type,  std::move of the variable 'rw_settings' of the trivially-copyable type 'S3Settings::ReadWriteSettings' has no effect. 
Reference:  https://clang.llvm.org/extra/clang-tidy/checks/performance-move-const-arg.html